### PR TITLE
Populate Dashboard numbers from GraphQL query

### LINKED ISSUE #162: Define where the data will be loaded for views like Dashboard
Follow up from #149 

When browsing `/#/dashboard`, the user will initiate a Route to a view. The view is handled by a component, in this case `src/views/Dashboard.vue`.

At the moment, `Dashboard.vue` is initiating a polling to retrieve data from GraphQL for the query. The design sketch document has what is in `GScan.vue` in the place of what we have in the `Drawer.vue` at the moment.

image

Once we move `GScan.vue` there, both components (`Dashboard.vue` and `GScan.vue`) will be displayed at the same time. This issue is for discussion around choosing when and how to fetch the data for components in Cylc UI.

### PR DESCRIPTION
**Pull request by kinow**
These changes close #162 also related to #94 

Let's add data to the dashboard :-)

**Requirements check-list**
- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.
- [x] Contains logically grouped changes (else tidy your branch by rebase).
- [x] Does not contain off-topic changes (use other PRs for other changes).
- [x] Does not need tests (why? adding some code to the view, but no dashboard component yet, could be covered by an e2e eventually).
- [x] Appropriate change log entry included.
- [x] No documentation update required.

### CODE REVIEW

**Comment by kinow** [comment]
image

File: src/styles/cylc/_dashboard.scss [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    width: auto;
+    height: auto;
+  }
+  .v-list__tile__action, .v-list__tile__avatar {
+    min-width: auto;
+    margin-left: -10px;
+  }
+
+  /* otherwise avatar does not left align with component above,
+     !important as the attributes are set in the tag element -->
```
Comment (oliver-sanders) [inline src/styles/cylc/_dashboard.scss]: erroneous end comment?

**Comment by oliver-sanders** [comment]
@datamel Once you have got the end-to-end system setup this might be a good first issue to look at.

File: src/styles/cylc/_dashboard.scss (hunk 1 above)
Comment (kinow) [reply src/styles/cylc/_dashboard.scss]: I'm impressed somehow it appears to be parsed correctly? Even `vim` is interpreting the next line as non-comment.

image

Comment (kinow) [reply src/styles/cylc/_dashboard.scss]: Fixed.

**Comment by datamel** [comment]
> 
> 
> @datamel Once you have got the end-to-end system setup this might be a good first issue to look at.
Yep @oliver-sanders , I'll have a good look once I've got my setup sorted - have just tried getting set up with a fresh conda environment but still having problems. I'll try and use your script from riot tomorrow and see if I can get it working.
Currently it is complaining about "vue-cli-service: command not found" when I run the build:watch in cylc-ui. Not sure if I need to do a specific installation of that.

**Pull request merged**